### PR TITLE
Enable relative protocol witness tables on all mixin_stdlib_minimal configurations

### PR DESCRIPTION
**Pull request by aschwaighofer**
So that we test the relative protocl witness table config on a bot

### CODE REVIEW

**Comment by aschwaighofer** [comment]
@swift-ci smoke test

**Comment by aschwaighofer** [comment]
preset=stdlib_S_standalone_minimal_macho_x86_64,build,test
@swift-ci please test with toolchain and preset

File: utils/build-presets.ini [hunk 1]
```diff
@@ -2689,6 +2689,8 @@ swift-freestanding-triple-name=macosx11.0
 swift-freestanding-module-name=macos
 swift-freestanding-archs=x86_64
 build-swift-private-stdlib=0
+swift-stdlib-use-relative-protocol-witness-tables=1
```
Comment (kubamracek) [inline utils/build-presets.ini:2692]: Can this be instead done in the parent of all the "minimal footprint" settings, i.e. in `mixin_stdlib_minimal`?

Comment (aschwaighofer) [reply utils/build-presets.ini:2692]: mixin_stdlib_minimal might be referenced by configurations that we don't intended to build with that feature enabled

Comment (aschwaighofer) [reply utils/build-presets.ini]: We can update those configurations if they pop up

**Comment by aschwaighofer** [comment]
@swift-ci smoke test

**Review by kubamracek — APPROVED** [review]

**Comment by aschwaighofer** [comment]
preset=stdlib_S_standalone_minimal_macho_x86_64,build,test
@swift-ci please test with toolchain and preset

**Pull request merged**